commons/marmotta-sesame-tools/marmotta-model-vocabs/pom.xml

```diff
@@ -26,7 +26,7 @@
     </parent>
 
     <artifactId>marmotta-model-vocabs</artifactId>
-    <packaging>jar</packaging>
+    <packaging>bundle</packaging>
```

Comment:
`marmotta-model-vocabs` is used by both `ldclient-provider-facebook` (https://git.io/vKZb7) and `ldclient-provider-ldap` (https://git.io/vKZNf). A compelling alternative would be to embed that dependencies in the relevant ldclient-provider-* bundles.